docs/api-reference/instances/attribute.md

```diff
@@ -37,9 +36,3 @@ print(label:GetAttribute("Ammo")) -- 10
 ```
 
 -----
-
```

Comment:
I didn't mean to imply the entire section was redundant here; just that it would have sufficed to have one sentence instead of two;

'This special key runs at the `self` stage'

as opposed to:

'This special key runs at the `self` stage. Due to the fact that it operates on the instance itself.'

Reply:
OH LOL my bad